ccloud-observability/monitoring_configs/prometheus/prometheus.template.yml

```
@@ -41,16 +41,24 @@ scrape_configs:
         labels:
           env: 'dev'
 
-  - job_name: "ccloud-exporter"
+  - job_name: 'confluent-cloud'
```

Quotes not needed here
```suggestion
  - job_name: confluent-cloud
```